src/test/java/hudson/plugins/performance/PerformancePipelineTest.java

```diff
@@ -0,0 +1,69 @@
+package hudson.plugins.performance;
+
+import hudson.model.Result;
+import hudson.slaves.DumbSlave;
+import org.apache.commons.io.IOUtils;
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.jenkinsci.plugins.workflow.job.WorkflowRun;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runners.model.Statement;
+import org.jvnet.hudson.test.BuildWatcher;
+import org.jvnet.hudson.test.RestartableJenkinsRule;
+
+import java.net.URL;
+
+public class PerformancePipelineTest  {
+    @ClassRule
+    public static BuildWatcher buildWatcher = new BuildWatcher();
+
+    @Rule
+    public RestartableJenkinsRule story = new RestartableJenkinsRule();
+
+    @Test
+    public void bztSmokeTests() throws Exception {
+        String path = getClass().getResource("/performanceTest.yml").getPath();
+        final String bztParams =  path + ' ' + "-o modules.jmeter.plugins=[] -o services=[] -o modules.jmeter.version=3.1 -o modules.jmeter.path=";
+
+        story.addStep(new Statement() {
+            public void evaluate() throws Throwable {
+                DumbSlave s = story.j.createOnlineSlave();
+                s.setLabelString("test performance test ");
+                WorkflowJob p = story.j.createProject(WorkflowJob.class, "demo");
+                p.setDefinition(new CpsFlowDefinition(
```

Comment:
So my guess here is that this is not properly running on the agent. Gonna see if I can rework things further.